src/test/emulators/storage/README.md

```diff
@@ -0,0 +1,43 @@
+# Firebase Storage emulator
```

Comment:
thanks for the README! could you submit this in a separate PR? I don't think it's related to this name issue